pynblint/nb_linting.py

```diff
@@ -282,6 +291,12 @@ def cells_too_long(notebook: Notebook) -> List[Cell]:
         "a few more Markdown cells.",
         linting_function=too_few_MD_cells,
     ),
+    LintDefinition(
+        slug="Not_renamed_duplicate_notebook",
```

Comment:
We use only lowercase letters for slug names.

Moreover, words are separated with dashes in Pynblint slugs, not underscores.